src/views/Peers.tsx

```diff
@@ -154,99 +181,141 @@ export const Peers = () => {
         });
     }
 
+    const setUpdateGroupsVisible = (peerToAction:Peer, status:boolean) => {
+        if (status) {
+            dispatch(peerActions.setPeer({...peerToAction}))
+            dispatch(peerActions.setUpdateGroupsVisible(true))
+            return
+        }
+        dispatch(peerActions.setPeer(null))
+        dispatch(peerActions.setUpdateGroupsVisible(false))
+    }
+
+    const renderPopoverGroups = (label: string, groups:Group[] | string[] | null, peerToAction:PeerDataTable) => {
+        const content = groups?.map((g,i) => {
+            const _g = g as Group
```

Comment:
Codacy has a fix for the issue: [Replace `············const·_g·=·g·as·Group` with `······const·_g·=·g·as·Group;`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
      const _g = g as Group;
```